sites/partners/src/pages/listings/[id]/edit.tsx

```diff
@@ -64,7 +64,9 @@ export async function getServerSideProps(context: { params: Record<string, strin
   let response
 
   try {
-    response = await axios.get(`${process.env.backendApiBase}/listings/${context.params.id}`)
+    response = await axios.get(`${process.env.backendApiBase}/listings/${context.params.id}`, {
+      headers: { passkey: process.env.API_PASS_KEY },
```

Comment:
Why is this required on this specific endpoint?

Reply:
I'm not sure I understand the question. Almost all endpoints on the api now require the passkey is sent across. If we don't send it here then the api will respond with an error and will prevent the user from making changes to listings